DifferenceKit.podspec

```diff
@@ -13,6 +13,7 @@ Pod::Spec.new do |spec|
   spec.license = { :type => 'MIT', :file => 'LICENSE' }
   spec.requires_arc = true
   spec.default_subspecs = 'Core', 'UIKitExtension'
+  spec.swift_version = "4.2"
```

Comment:
Shouldn't this be 5.0?

Reply:
👌

Reply:
Probably shouldn't 😂

Reply:
@Kaspik 
DifferenceKit should supports both Swift4.2 and 5.0.
 Xcode10.2 is able to build sources that are written in either Swift 5, Swift 4.2.